### ISSUE TITLE
Use data_table also in jupyterlab, not only in colab?

### ISSUE THREAD

**Issue by Nestak2** [googlecolab/colabtools#971]
@colaboratory-team  @blois  @katlyn-edwards I really love data_table, thanks for creating in for colab! But more often I need to use jupyterlab than colab and I wonder if I can easily install `google.colab.data_table` for jupyterlab? As an extension maybe? If not, are you considering making this possible in the future?

**Comment by blois** [comment]
We don't have any immediate plans but I do think this is a reasonable request. There is a similar discussion on https://github.com/nteract/data-explorer/issues/6 where I shared a prototype of a similar approach for nteract's Data Explorer.

**Comment by vivekpayasi** [comment]
@Nestak2 
I think you can use Qgrid in JupyterLab, which has similar functionality as `data_table` extension in Colab.
Here's the page: https://github.com/quantopian/qgrid

**Comment by Nestak2** [comment]
@vivekpayasi Thanks for the hint! I have already found Qgrid, but it is having continuous issues with the installation, therefore I use for now tubitv/jupyterlab-extensions: https://github.com/Tubitv/jupyterlab-extensions

**Comment by vivekpayasi** [comment]
@Nestak2 Oh wow! I didn't know that nteract's data-explorer was available as a JupyterLab extension. Thanks :)

**Comment by Nestak2** [comment]
@vivekpayasi Yes, and there is a convenient installation file, so you don't have to go yourself through all the needed packages. Still, I would like to see data_table for jupyterlab, it offered the best sorting functionality I have seen so far

**Comment by CourierKyn** [comment]
```
!pip install -q git+https://github.com/googlecolab/colabtools
%load_ext google.colab.data_table
```

just use this in your local Jupyter environment or jupyterlab
